src/components/PupilViews/PupilQuiz/PupilQuiz.view.tsx

```diff
@@ -162,11 +189,31 @@ const QuizInner = () => {
     </OakLessonBottomNav>
   );
 
+  const handleBackLinkClick = () => {
+    switch (currentSection) {
+      case "starter-quiz":
+        if (track.lessonActivityAbandonedStarterQuiz) {
+          track.lessonActivityAbandonedStarterQuiz(
```

Comment:
Again component coordinates will be useful here.

Reply:
oh will they